packages/client/components/TeamPrompt/TeamPromptTopBar.tsx

```diff
@@ -95,6 +95,7 @@ const TeamPromptTopBar = (props: Props) => {
         facilitatorUserId
         meetingSeries {
           id
+          title
```

Comment:
+1 why are we adding this?

Reply:
testing leftover, removed, good catch!

Reply:
done, forgot to push changes after James did the code review 🤦‍♂️